float32[4] ekf_output
float32[1] e_m
float32[1] e_m_dot
float32 u_bc
float32 v_bc
float32 time